pkgs/development/libraries/gmock/default.nix

```diff
@@ -1,15 +1,13 @@
-{ stdenv, fetchurl, unzip, cmake}:
-
+{ stdenv, cmake, callPackage }:
+let
+  source = callPackage ./source.nix { };
```

Comment:
Or maybe `fetchFromGitHub` directly, as I see the URL redirects to github anyway.